sdk/translation/azure-ai-translation-document/src/test/java/com/azure/ai/translation/document/TestData/test-glossary.csv

```diff
@@ -0,0 +1 @@
+test,test
```

Comment:
These files in `TestData` directory should be in `src/test/resources` directory.

Reply:
Moved the test files.